lib/calculate-modulo-256-via-x86-assembly.yml

```diff
@@ -15,5 +15,5 @@ rule:
     - and:
       - mnemonic: and
       - or:
-        - number: 0x800000FF
-        - number: 0xFF
+        - operand[1].number: 0x800000FF
+        - operand[1].number: 0xFF
```

Comment:
can you explain this change?

Reply:
trying to be more explicit here, but yeah, there can only be an immediate in the second operand, so I guess this is not useful here

Reply:
any preference? be explicit here or undo change and stick to `number`?